Dockerfile

```diff
@@ -6,14 +6,14 @@ ENV LC_CTYPE C.UTF-8
 
 FROM base as builder
 RUN apt-get update && apt-get install -y build-essential cmake libsparsehash-dev
-RUN mkdir -p /app/build
 COPY . /app/
 WORKDIR /app/build/
 RUN cmake -DCMAKE_BUILD_TYPE=Release .. && make -j 10
 
 FROM base as runtime
-RUN mkdir /app /index /input
 WORKDIR /app
+RUN groupadd -r qlever && useradd --no-log-init -r -u 1000 -g qlever qlever && chown qlever:qlever /app
```

Comment:
I'm not quite sure here, don't you want something like:
```
ARG UID=1000
RUN groupadd -r qlever && useradd --no-log-init -r -u $UID -g qlever qlever && chown qlever:qlever /app
```
?

And later, in the README use:
```docker run --build-arg UID=$(id -u)```

Reply:
@naetherm `--build-arg` only works during `docker build` and at that point one often doesn't know the target UID